weasel/about.py

```diff
@@ -0,0 +1,3 @@
+__version__ = "3.5.0"
+__projects__ = "https://github.com/explosion/projects"
+__projects_branch__ = "v3"
```

Comment:
`spacy`-specific stuff like this is intentionally ignored in this PR, correct?

Reply:
There are no tests running yet, it's all doing nothing.